File: app/src/main/java/de/bornholdtlee/snh/utils/DateUtils.java
package de.bornholdtlee.snh.utils;

import android.text.format.DateFormat;

public class DateUtils {

    private static final long ONE_MIN_IN_MILLI = 1000 * 60;
    public static final long FIFTEEN_MIN_IN_MILLI = ONE_MIN_IN_MILLI * 15;
    private static final long ONE_HOUR_IN_MILLI = ONE_MIN_IN_MILLI * 60;
    public static final long ONE_DAY_IN_MILLI = ONE_HOUR_IN_MILLI * 24;
    public static final long ONE_MONTH_IN_MILLI = ONE_DAY_IN_MILLI * 30;


    public String formatMillisToReadable(long updateTimestamp) {
        return DateFormat.format("dd.MM.yyyy HH:mm", new java.util.Date(updateTimestamp)).toString();
    }
}

File: app/src/main/java/de/bitb/astroskop/utils/DateUtils.kt
package de.bitb.astroskop.utils

import android.text.format.DateFormat

class DateUtils {


    fun formatMillisToReadable(updateTimestamp: Long): String {
        return DateFormat.format("dd.MM.yyyy HH:mm", java.util.Date(updateTimestamp)).toString()
    }

    companion object {

        private val ONE_MIN_IN_MILLI = (1000 * 60).toLong()
        val FIFTEEN_MIN_IN_MILLI = ONE_MIN_IN_MILLI * 15
        private val ONE_HOUR_IN_MILLI = ONE_MIN_IN_MILLI * 60
        val ONE_DAY_IN_MILLI = ONE_HOUR_IN_MILLI * 24
        val ONE_MONTH_IN_MILLI = ONE_DAY_IN_MILLI * 30
    }
}
